### PR TITLE
HB-5368: Remove support for multiple banner loads for the same placement

### PR DESCRIPTION
**Pull request by daniel-barros**

### CODE REVIEW

File: Source/VungleAdapterBannerAd.swift [hunk 1]
```diff
@@ -159,6 +147,9 @@ final class VungleAdapterBannerAd: VungleAdapterAd, PartnerAd {
             // Non-programmatic
             try VungleSDK.shared().addAdView(to: inlineView, withOptions: [:], placementID: request.partnerPlacement)
         }
+        
+        // Mark ad as displayed so we know if we need to mark it as finished later on `invalidate()`.
+        adWasDisplayed = true
```
Comment (CB-HaoxinLi) [inline Source/VungleAdapterBannerAd.swift:152]: `invalidate()` does not set `adWasDisplayed` back to `false`. Is it expected behavior?

```
func invalidate() throws {
        if adWasDisplayed {
            finishDisplayingVungleAd()
        }
        log(.invalidateSucceeded)
    }
```

Comment (daniel-barros) [reply Source/VungleAdapterBannerAd.swift:152]: yeah, invalidate is called when the ad is discarded, just before it is deallocated. So there's no need to clean up state at that point

**Comment by daniel-barros** [comment]
@CB-HaoxinLi @kendallrogers can I get a reapproval?

**Review by ricealexanderb — APPROVED** [review]
LGTM

**Review by kendallrogers — APPROVED** [review]

**Pull request merged**